ix-dev/community/windmill/ix_values.yaml

```diff
@@ -0,0 +1,29 @@
+images:
+  image:
+    repository: ghcr.io/windmill-labs/windmill
+    tag: "1.485.3"
+  postgres_16_image:
+    repository: postgres
+    tag: "16"
+  caddy_image:
+    repository: ghcr.io/windmill-labs/caddy-l4
+    tag: "latest"
+  lsp_image:
+    repository: ghcr.io/windmill-labs/windmill-lsp
+    tag: "1.485.3"
+
+consts:
+  windmill_server_container_name: windmill_server
+  windmill_worker_container_name: windmill_worker
+  windmill_worker_native_container_name: windmill_worker_native
+  lsp_container_name: lsp
+  caddy_container_name: caddy
+  perms_container_name: permissions
+  postgres_container_name: postgres
+  data_path: /data
+  db_user: windmill
+  db_name: windmill
+  ssl_cert_path: /certs/ssl.crt
+  ssl_key_path: /certs/ssl.key
+  pg_run_user: 999
```

Comment:
those are unused, or if they are, they shouldnt be used. pg dependency sets those automatically